Lib/test/test_ntpath.py

```diff
@@ -270,6 +277,12 @@ def test_normpath(self):
         tester("ntpath.normpath('//server/share/../..')",  '\\\\server\\share\\')
         tester("ntpath.normpath('//server/share/../../')", '\\\\server\\share\\')
 
+        # gh-96290: don't normalize partial/invalid UNC drives
+        tester("ntpath.normpath('\\\\foo\\bar')", '\\\\foo\\bar')
```

Comment:
This is a valid UNC path. Maybe include a commented out test case that depends on fixing `nt._path_normpath()`.

```suggestion
        # BUGBUG: nt._path_normpath() needs to be fixed to match
        # ntpath.splitdrive() for an empty share. Skip this for now.
        # tester("ntpath.normpath('\\\\foo\\\\')", '\\\\foo\\')
```

Reply:
I'll log a bug for this after this PR is merged!